typescript/infra/fork.sh

```diff
@@ -28,15 +28,15 @@ done
 set -e
 
 echo "=== Run $MODULE checker against forked $ENVIRONMENT ==="
-DEBUG=hyperlane:* yarn ts-node ./scripts/check-deploy.ts -e $ENVIRONMENT -f $FORK_CHAIN -m $MODULE
```

Comment:
There's still benefit to having the `DEBUG` env set here in case people run locally right?

Reply:
yeah I guess if they want it easy to prefix?